server_admin/topics/clients/con-advanced-settings.adoc

```diff
@@ -1,72 +1,84 @@
 [id="con-advanced-settings_{context}"]
 ==== Advanced Settings
-There are 
+Advanced settings also available for OIDC Clients.
```

Comment:
How about: When you click *Advanced Settings*, additional fields appear.

Reply:
Going with "are displayed" as the style guide does not like "appear".